src/zolltools/nemsis/locationcodes.py

```diff
@@ -36,3 +43,54 @@ def to_description(code, default=None):
         return "not-found"
 
     return description
+
+
+def _list():
+    """Lists location code groupings"""
+
+    storage_dir = Path.cwd().joinpath(_STORAGE_FOLDER_NAME)
+    paths = storage_dir.glob(f"*{_STORAGE_FOLDER_EXT}")
+    names = sorted(
+        [strtools.removesuffix(path.name, _STORAGE_FOLDER_EXT) for path in paths]
+    )
+
+    print("\n".join(names))
+
+
+def _validate():
+    """Validates a location code grouping"""
+
+
+def _init():
+    """Initializes a folder for location code groupings"""
+
+    storage_dir = Path.cwd().joinpath(_STORAGE_FOLDER_NAME)
+    storage_dir.mkdir(exist_ok=True)
```

Comment:
`exist_ok=True` is used so that an error is not raised if `python -m locationcodes init` is run twice.